archunit/src/test/java/com/tngtech/archunit/core/domain/JavaClassTest.java

```diff
@@ -519,12 +519,20 @@ public void direct_dependencies_from_self_by_inheritance_through_generic_supercl
 
         JavaClass javaClass = importClassWithContext(Child.class);
 
+        String genericSuperclass = Base.class.getName() + "<"
```

Comment:
Yeah, that's kind of true :thinking: I'll relax the description assertion, because in the end it's only used to make sure we have one dependency per target type. The exact description is not the issue here, you're right, so we shouldn't assert it too strictly.